src/alignment.js

```diff
@@ -121,7 +121,7 @@ var alignWithDOM = function(nodeName, key, statics) {
 /**
  * Clears out any unvisited Nodes, as the corresponding virtual element
  * functions were never called for them.
- * @param {!Element} node
+ * @param {!Node} node
```

Comment:
It could be a DocumentFragment, which is not an Element. Other than patching literal document fragments, this is also useful for patching shadow roots. We could be a little more explicit and say `{!Element|!DocumentFragment}`.